app/controllers/staff/organisations_controller.rb

```diff
@@ -32,8 +32,32 @@ def create
     end
   end
 
+  def edit
+    @organisation = policy_scope(Organisation).find(id)
+    authorize @organisation
+  end
+
+  def update
+    @organisation = policy_scope(Organisation).find(id)
+    authorize @organisation
+
+    @organisation.assign_attributes(organisation_params)
+
+    if @organisation.valid?
```

Comment:
I think this is good for consistency with controller actions that use a service to perform the update, even if the standard way takes less lines, e.g.

```ruby
if @organisation.update(organisation_params)
  flash[:notice] = ...
  ...
else
  render :edit
end 
```